### PR TITLE
Added Izura-2005-499 norms on Spanish words

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
Notes about the data:
- the words were collected in a written category generation test with Spanish native speakers
- the ratings were conducted while participants knew the semantic category of a word which disambiguates some of the meanings, e.g., _sharp_ (category: intelligence) refers to "klar" instead of "scharf".

Notes about the mapping:
- I introduced some new concepts. Please have a look at the definitions because I made them myself.
- We don't have the concept LOVE as a noun but I suppose that most of the other mappings refer to either the noun or the verb.
- there are two forms of **double mapping**:
1. HEAD is listed in the "intelligence" and "body part" category
2. some English translations occur multiple times, e.g., _trousers_/_pants_, _bag_. I tried to select the dominant meaning for the mapping. @LinguList could you please check which one is the right one?

### CODE REVIEW

File: concepticondata/references/references.bib [hunk 1]
```diff
@@ -2751,3 +2751,29 @@ @article{Lynott2009
   year={2009},
   publisher={Springer}
 }
+
+
+@article{Izura2005,
+  title={Category norms for 500 Spanish words in five semantic categories},
+  author={Izura, Cristina and Hernandez-Munoz, Natividad and Ellis, Andrew W},
```
Comment (chrzyki) [inline concepticondata/references/references.bib]: W.

File: concepticondata/references/references.bib [hunk 2]
```diff
@@ -2751,3 +2751,29 @@ @article{Lynott2009
...
+
+@book{Alameda1995,
+  title={Diccionario de frecuencias de las unidades linguisticas del castellano},
+  author={Alameda, Jose Ramon and Cuetos, Fernando},
+  year={1995},
+  publisher={Univ. de Oviedo, Servicio de Publ.}
+}
+
+@phdthesis{Hernandez2002,
+  author       = {Hernandez-Munoz, N.},
```
Comment (chrzyki) [inline concepticondata/references/references.bib]: First name?

**Comment by chrzyki** [comment]
Thanks, just remarked minor things. :)

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-13	13	Cabeza	head	HEAD	1256	intelligence	5.84	6.6	4.2	6.72	5.958	416	3
+Izura-2005-499-14	14	Capacidad	capacity			intelligence	5.92	6.16	8.13	3.48	4.713	114	4
+Izura-2005-499-15	15	Carisma	charisma			intelligence	3.76	4.44	11.32	3.24	1.47	0	3
+Izura-2005-499-16	16	Carrera	career			intelligence	4.48	6.52	8.84	4.72	2.268	77	3
+Izura-2005-499-17	17	Cerebro	brain	BRAIN	1583	intelligence	6.12	5.4	7.04	5.48	23.892	87	3
+Izura-2005-499-18	18	Ciencia	science			intelligence	5.16	5.32	8.36	3.52	3.037	203	2
+Izura-2005-499-19	19	Científico	scientist			intelligence	5.12	4.64	8.52	6.24	2.268	85	4
+Izura-2005-499-20	20	Comprender	to understand	UNDERSTAND	1536	intelligence	6	6	6.52	3.6	2.972	67	3
+Izura-2005-499-21	21	Concentración	concentration			intelligence	5.2	6	8	3.64	1.635	40	4
+Izura-2005-499-22	22	Conocer	to know	KNOW (SOMEBODY)	2248	intelligence	5.56	5.88	5.92	3.36	3.845	104	3
```
[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: The differences are a big more finegrained here, as you have *conocer* and *saber*, as in German, where you can also say "ich kenne ein Buch", while English merges both. So key question is: how was "know somebody" so far treated in our Spanish elicitation glosses. I assume: the preference was given to "know (somebody)", although this is not entirely correct, of course.

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-77	77	Razón	reason	REASON	2968	intelligence	5.76	5.44	8.6	2.84	1.695	275	2
+Izura-2005-499-78	78	Razonamiento	reasoning			intelligence	6.6	5	10.32	2.96	1.602	8	5
+Izura-2005-499-79	79	Razonar	to reason			intelligence	6.64	5.16	10.04	3.04	1.763	5	3
+Izura-2005-499-80	80	Recordarto	remember	REMEMBER	707	intelligence	4.64	5.12	7.16	3.32	2.805	83	3
+Izura-2005-499-81	81	Recuerdo	memory	MEMORY	1018	intelligence	3.52	5.84	7.4	4.08	1.418	177	3
+Izura-2005-499-82	82	Reflexión	reflection			intelligence	5.92	4.64	9.8	3.16	1.488	41	3
+Izura-2005-499-83	83	Representación	representation			intelligence	3.48	3.44	9.72	3.72	2.235	46	5
+Izura-2005-499-84	84	Resolver	to solve			intelligence	6.04	4.76	8.68	3.12	1.627	30	3
+Izura-2005-499-85	85	Responsabilidad	responsibility			intelligence	3.48	5.52	9.04	2.92	3.498	39	6
+Izura-2005-499-86	86	Respuesta	answer			intelligence	4.6	5.28	6.32	3.44	1.527	95	3
```
[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: Is the verbal expression. Would be *contestar*.

File: concepticondata/conceptlists/Izura-2005-499.tsv (hunk 3 above)
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: _saber_ was mapped to KNOW (SOMETHING) in the past. 

I've tried to differentiate the two meanings (_saber_ is also in the list!) and I found that _conocer_ can also mean "meet somebody".

[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: BTW: [Gabelentz has "know" = "wissen"](https://concepticon.clld.org/values/Gabelentz-1891-120-120), this is surely wrong. Could you, @AnnikaTjuka link this to "know (something)" in your PR? Then I do not need to make an issue extra for this.

[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: > BTW: [Gabelentz has "know" = "wissen"](https://concepticon.clld.org/values/Gabelentz-1891-120-120), this is surely wrong. Could you, @AnnikaTjuka link this to "know (something)" in your PR? Then I do not need to make an issue extra for this.

Sure!

[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: There is no precedence for this case in Spanish. So there are two possibilities:

1. link to "know (somebody or somthing)" to go for the most prototypical meaning (we already had this concept added)
2. make an issue that we need to re-link all know instances, with at least three meanings: 
    - know (be knowledgeable of something): "I know that X"
    - know (be acquainted with something): "I know a book"
    - know (a person): "I know her"
    - know (how to): "I know how to swim"
    - maybe more meaning shades I missed here?

In principle, we can even leave it at 1 for the time being, but I have the feeling we have to review the "know" concepts to make sure they are all properly linked. What I mention here is just my current impression, much to evolve by digging a bit deeper, I assume.

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 5]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-338	338	Faringe	pharynx	PHARYNX	3713	body part	3.04	2.6	9.12	4.24	2.642	1	3
+Izura-2005-499-339	339	Femur	femur			body part	3.36	2.48	9.24	5.08	2.564	0	2
+Izura-2005-499-340	340	Frente	forehead	FOREHEAD	123	body part	5.56	4.96	4.88	6.48	5.747	375	2
+Izura-2005-499-341	341	Gemelo	calf muscle			body part	4.08	4.76	8.68	6.12	3.222	3	3
+Izura-2005-499-342	342	Glúteo	gluteus			body part	4.24	3.88	8.92	6.48	4.044	1	3
+Izura-2005-499-343	343	Hígado	liver	LIVER	1224	body part	4.16	4	7.92	5.56	7.299	12	3
+Izura-2005-499-344	344	Hombro	shoulder	SHOULDER	1482	body part	5.72	5.32	4.68	6.8	29.002	45	2
+Izura-2005-499-345	345	Hueso	bone	BONE	1394	body part	4.96	5.08	4.4	6.68	5.329	22	2
+Izura-2005-499-346	346	Húmero	humerus			body part	2.96	2.4	8.6	4.6	1.576	0	3
+Izura-2005-499-347	347	Incisivo	incisor			body part	2.8	2.44	8.6	5	93.5	2	4
```
[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: Only the English word, the Spanish means "einschneidend" (as in "einschneidende Maßnahme").

[the remark this answers is not in the document]
Comment (chrzyki) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: Even here with the category 'body part'?

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 6]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-230	230	Encimera	worktop			furniture	4.48	5.12	7.79	6.4	9.094	0	4
+Izura-2005-499-231	231	Entradita	entrance hall  			furniture	2.68	2.92	9.19	5.44	2.156	0	4
+Izura-2005-499-232	232	Escabel	footstool			furniture	2	1.16	11.18	2.88	0.641	2	3
+Izura-2005-499-233	233	Escritor	desk	TABLE	1775	furniture	5.88	5.8	7.46	6.76	28.341	14	4
+Izura-2005-499-234	234	Escurreplatos	plate rack			furniture	3.24	4.52	7.88	6.68	0.842	0	5
+Izura-2005-499-235	235	Espejo	mirror	MIRROR	1171	furniture	4.56	6.32	3.76	6.92	13.704	74	3
+Izura-2005-499-236	236	Estante	shelf	SHELF	745	furniture	5.52	5.04	5.46	6.8	0.699	4	3
+Izura-2005-499-237	237	Estantería	shelves			furniture	6.28	6.24	5.28	6.88	44.794	5	5
+Izura-2005-499-238	238	Flexo	reading lamp			furniture	3.52	6.12	8.2	6.32	1.925	2	2
+Izura-2005-499-239	239	Fogón	ring	RING	1374	furniture	2.64	3.16	7.71	6.08	0.679	4	2
```
Comment (LinguList) [inline concepticondata/conceptlists/Izura-2005-499.tsv]: This is not a ring. Suggest to unlink it.

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 7]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-268	268	Portalibros	book shelf   			furniture	2.84	2.28	11.15	5.2	0.641	0	4
+Izura-2005-499-269	269	Posaplatos	place mat			furniture	2.04	2.36	11.64	5.36	0.679	0	4
+Izura-2005-499-270	270	Puerta	door	DOOR	1567	furniture	3.48	6.56	4.24	7	4.876	301	2
+Izura-2005-499-271	271	Puf	pouffe			furniture	3	2.48	10.86	5.08	3.812	0	1
+Izura-2005-499-272	272	Pupitre	school desk			furniture	5.44	3.68	6	6.76	1.752	9	3
+Izura-2005-499-273	273	Radio	radio	RADIO	1315	furniture	2.24	5.76	5.68	6.96	1.787	75	2
+Izura-2005-499-274	274	Recibidor	hall     			furniture	3.88	3.16	8.72	5.96	2.872	7	4
+Izura-2005-499-275	275	Reloj	clock	CLOCK	1323	furniture	2.52	6.24	4.92	7	2.519	43	2
+Izura-2005-499-276	276	Repisa	shelf	SHELF	745	furniture	3.6	4.04	7.84	6.64	1	3	3
+Izura-2005-499-277	277	Retrete	loo			furniture	1.8	3.36	8.64	7	1.261	14	3
```
Comment (LinguList) [inline concepticondata/conceptlists/Izura-2005-499.tsv]: we have a concept set for toilet, right? This is what this refers to.

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 8]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-280	280	Ropero	wardrobe	WARDROBE	3106	furniture	6.08	3.32	9.84	6.48	2.893	6	3
+Izura-2005-499-281	281	Secreter	writing desk			furniture	4.44	1.36	16.89	4.12	1.215	0	3
+Izura-2005-499-282	282	Silla	chair	CHAIR	1476	furniture	6.8	6.72	4.64	7	78.805	58	2
+Izura-2005-499-283	283	Sillón	armchair			furniture	6.76	5.84	5	7	61.626	31	2
+Izura-2005-499-284	284	Sinfonier	chest of drawers			furniture	4.2	1.64	13.69	3.24	3.479	0	3
+Izura-2005-499-285	285	Sofá	sofa	SOFA	3099	furniture	6.8	6.36	6.24	7	56.795	4	2
+Izura-2005-499-286	286	Taburete	stool	STOOL	2058	furniture	5.72	4.48	6.08	6.96	18.16	2	4
+Izura-2005-499-287	287	Taquillón	traditional Spanish table			furniture	4.84	2.32	9.95	5.52	2.278	0	3
+Izura-2005-499-288	288	Televisión	television	TELEVISION	1788	furniture	4	6.24	4.35	6.8	14.852	85	4
+Izura-2005-499-289	289	Televisor	television			furniture	2.88	5.76	6.52	6.96	2.819	25	4
```
Comment (LinguList) [inline concepticondata/conceptlists/Izura-2005-499.tsv]: if they have two separate elicitation glosses, you are allowed to link to the same concept set. These mergers result from synonymy in the concept list.

File: concepticondata/conceptlists/Izura-2005-499.tsv (hunk 3 above)
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: I've created a new issue #700 but linked the concept to KNOW (SOMEBODY or SOMETHING) for now. Note that the problem was also mentioned in #607.

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 9]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-292	292	Tresillo	sofa	SOFA	3099	furniture	6.04	2.88	9.96	5.96	8.622	4	3
+Izura-2005-499-293	293	Tumbona	deck chair			furniture	4.32	2.04	9.72	6.52	1.454	2	3
+Izura-2005-499-294	294	Váter	toilet	TOILET	219	furniture	1.68	5.8	5.36	6.56	2.025	1	2
+Izura-2005-499-295	295	Ventana	window	WINDOW	1375	furniture	2.2	6.56	4.36	6.96	3.336	120	3
+Izura-2005-499-296	296	Video	video			furniture	1.84	5.52	10.6	7	1.808	30	3
+Izura-2005-499-297	297	Vidriera	stained-glass window			furniture	1.72	2.36	10.36	6.56	1.463	5	3
+Izura-2005-499-298	298	Vitrina	glass cabinet			furniture	4.12	2.64	10.6	6.44	2.906	6	3
+Izura-2005-499-299	299	WC	toilet 			furniture	1.88	3.64	9.69	6.68	2.658	0	2
+Izura-2005-499-300	300	Zapatero	shoe rack   			furniture	4.56	2.6	7.24	6.2	8.043	0	4
+Izura-2005-499-301	301	Abdomen	abdomen			body part	4.92	3.32	8.68	5.08	4.195	5	3
```
Comment (LinguList) [inline concepticondata/conceptlists/Izura-2005-499.tsv]: we tend to treat abdomen as "belly".

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 10]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-363	363	Ojo	eye	EYE	1248	body part	6.64	6.24	4.08	6.76	60.1	96	2
+Izura-2005-499-364	364	Ombligo	bellybutton	BELLYBUTTON	3715	body part	4.68	4.12	5.4	6.96	8.3	8	3
+Izura-2005-499-365	365	Omóplato	shoulder blade	SHOULDERBLADE	172	body part	3.48	2.16	9.24	5.52	2.6	1	4
+Izura-2005-499-366	366	Oreja	ear	EAR	1247	body part	6.56	4.92	4.32	6.84	48.1	35	3
+Izura-2005-499-367	367	Ovario	ovary	OVARY	3716	body part	3.32	3.16	10.48	4.44	1.6	1	3
+Izura-2005-499-368	368	Pancreas	pancreas	PANCREAS	3252	body part	3.12	1.72	10.8	4.04	4.7	0	3
+Izura-2005-499-369	369	Pantorrilla	calf	CALF OF LEG	498	body part	4.68	2.4	8.92	5.92	1.4	2	4
+Izura-2005-499-370	370	Párpado	eyelid	EYELID	17	body part	4.16	3.36	7.92	6.68	2.4	3	3
+Izura-2005-499-371	371	Pecho	breast	BREAST	1402	body part	6.12	4.08	6.76	6.56	21.7	94	2
+Izura-2005-499-372	372	Pechos	breasts			body part	5.96	4.96	7.52	6.72	2.2	34	2
```
Comment (LinguList) [inline concepticondata/conceptlists/Izura-2005-499.tsv]: same here, link to BREAST.

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 11]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-427	427	Cordero	lamb	LAMB	820	animal	5.76	4.28	3.84	6.8	7.306	14	3
+Izura-2005-499-428	428	Cucaracha	cockroach	COCKROACH	1575	animal	2.8	3.2	4.56	6.84	3.545	2	4
+Izura-2005-499-429	429	Cuervo	crow	CROW	913	animal	3.56	2.76	4.88	6.64	10.398	4	2
+Izura-2005-499-430	430	Culebra	snake	SNAKE	730	animal	3.84	3.24	4.68	6.84	7.098	1	3
+Izura-2005-499-431	431	Delfín	dolphin	DOLPHIN	1479	animal	5.44	3.2	5.4	6.88	8.164	2	2
+Izura-2005-499-432	432	Elefante	elephant	ELEPHANT	1290	animal	6.32	3.36	4.36	6.92	21.565	9	4
+Izura-2005-499-433	433	Foca	seal	SEAL	651	animal	4.84	3.52	5	6.88	3.533	1	2
+Izura-2005-499-434	434	Gallina	hen	HEN	1514	animal	5.88	4.88	3.36	6.88	25.464	19	3
+Izura-2005-499-435	435	Gallo	rooster	ROOSTER	1511	animal	5.56	4.48	3.52	6.92	11.802	16	2
+Izura-2005-499-436	436	Gamba	prawn			animal	2.56	4.52	5.72	6.76	3.811	1	2
```
Comment (LinguList) [inline concepticondata/conceptlists/Izura-2005-499.tsv]: 238 PRAWNS

File: concepticondata/conceptlists/Izura-2005-499.tsv [hunk 12]
```diff
@@ -0,0 +1,500 @@
...
+Izura-2005-499-447	447	Hamster	hamster	HAMSTER	3147	animal	3.92	3.6	7.56	6.68	8.385	1	2
+Izura-2005-499-448	448	Hiena	hyena	HYENA	1799	animal	3.08	2.28	7.16	6.24	7.051	1	2
+Izura-2005-499-449	449	Hipopótamo	hippopotamus	HIPPOPOTAMUS	3145	animal	4.44	2.28	6.16	6.84	10.631	2	5
+Izura-2005-499-450	450	Hormiga	aunt			animal	4.28	3.68	4.64	6.96	9.531	5	3
+Izura-2005-499-451	451	Iguana	iguana	IGUANA	1578	animal	3.72	1.88	9.16	5.72	3.109	7	3
+Izura-2005-499-452	452	Jabalí	wild boar	WILD BOAR	3154	animal	5.28	2.32	8.32	6.68	12.549	5	3
+Izura-2005-499-453	453	Jirafa	giraffe	GIRAFFE	3089	animal	5.4	2.16	6.48	7	18.874	1	3
+Izura-2005-499-454	454	Koala	koala			animal	4.52	1.68	9	6.67	4.209	0	3
+Izura-2005-499-455	455	Lagartija	lizard	LIZARD	632	animal	4.04	2.64	5.64	6.68	6.384	4	4
+Izura-2005-499-456	456	Lagarto	lizard			animal	3.8	2.04	6.76	6.32	4.21	4	3
```
Comment (LinguList) [inline concepticondata/conceptlists/Izura-2005-499.tsv]: LIZARD

File: concepticondata/conceptlists/Izura-2005-499.tsv (hunk 5 above)
[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: it is strange.

[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: Yes, this refers to Schneidezahn [see examples here](https://www.linguee.de/spanisch-deutsch/uebersetzung/dientes+incisivos.html).

File: concepticondata/conceptlists/Izura-2005-499.tsv (hunk 8 above)
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: Okay, that's good to know as this is also relevant for another list I'm preparing.

File: concepticondata/conceptlists/Izura-2005-499.tsv (hunk 3 above)
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: There is no concept  KNOW (SOMEBODY OR SOMETHING) in Concepticon. Is it possible that only the description was adapted. For KNOW:

_To have met and recognize somebody or to be certain or sure about something._

[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Izura-2005-499.tsv]: Ah, sorry. The concept is called KNOW and it includes "somebody or something".

**Comment by AnnikaTjuka** [comment]
I edited this list as well as updated the mappings in Calero-2002 and Gabelentz-1891. I also fixed the references.

**Comment by AnnikaTjuka** [comment]
I ran `concepticon make_linkdata` to recreate several mappings and pushed them into this branch, see #704 .

@LinguList If it is ok with you, I would merge the PR into the master. I edited the list according to the requested changes (https://github.com/concepticon/concepticon-data/pull/699#issuecomment-567904888)

**Comment by LinguList** [comment]
yes, please do so.

**Pull request merged**